### ISSUE TITLE
Move registrations and registry information to IANA Considerations (Trac #97)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#97]
Since we are required to have an IANA Considerations section,
it should be used to house the type registrations and other
documented registries (as opposed to leaving them in appendices).

This should be done during the next set of reorg changes.

Migrated from https://trac.ietf.org/ticket/97
```json
{
    "status": "closed",
    "changetime": "2008-07-31T20:15:04",
    "_ts": "1217535304000000",
    "description": "Since we are required to have an IANA Considerations section,\nit should be used to house the type registrations and other\ndocumented registries (as opposed to leaving them in appendices).\n\nThis should be done during the next set of reorg changes.\n",
    "reporter": "fielding@gbiv.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-02-05T01:15:36",
    "component": "p1-messaging",
    "summary": "Move registrations and registry information to IANA Considerations",
    "priority": "",
    "keywords": "",
    "milestone": "04",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080601T12:20:48: julian.reschke@gmx.de commented:

> Can we make this a bit more specific?
> 
> We didn't have registrations for headers (#40) and status codes (#59) so far; they have been added now ([253] and [255]).
> 
> The method registry is still a TODO (#72).
> 
> So is this just about the MIME type registrations (Part1, Appendix A, and Part5, Appendix A)?
>

**Comment by mnot** [comment]
Trac update at 20080717T14:22:19: julian.reschke@gmx.de changed milestone from "03" to "04"

**Comment by mnot** [comment]
Trac update at 20080729T16:54:08: 
* julian.reschke@gmx.de commented:

> Move Appendix A into the IANA considerations.


* julian.reschke@gmx.de changed component from "non-specific" to "p1-messaging"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080729T18:51:36: 
* julian.reschke@gmx.de commented:

> Fixed in [296]:
> 
> Resolve #97: Move media type registrations into IANA considerations section (closes #97).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20080731T20:15:04: julian.reschke@gmx.de commented:

> From [299]:
> 
> Re-gen HTML due to reference target having been moved (related to #97).
